src/_base/helm/app/templates/service/mailhog-relay/service.yaml

```
@@ -0,0 +1,11 @@
+{{ if .Values.docker.services.mailhog_relay.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.service: {{ .Values.resourcePrefix }}mailhog-relay
+  name: {{ .Values.resourcePrefix }}mailhog-relay
```

I guess technically to match the development environment:
```suggestion
  name: mailhog-relay
```